Assets/Mirror/Components/Discovery/NetworkDiscoveryBase.cs

```diff
@@ -32,8 +32,8 @@ public abstract class NetworkDiscoveryBase<Request, Response> : MonoBehaviour
         protected int serverBroadcastListenPort = 47777;
 
         [SerializeField]
-        [Tooltip("Time in seconds between multi-cast messages")]
-        [Range(1, 60)]
+        [Tooltip("Time in seconds between multi-cast messages. Use a negative value to disable automatic broadcasting.")]
+        [Range(-1, 60)]
```

Comment:
better to add a bool.
it's not obvious what negative time is (doesn't really exist in physics)

Reply:
You're right, I changed it to a bool